app.rb

```diff
@@ -0,0 +1,142 @@
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+
+def list_books
+  puts 'List of books:'
+  @books.each { |book| puts "Title: \"#{book.title}\", Author: #{book.author}" }
+end
+
+def list_people
+  puts 'List of people:'
+  @people.each { |person| puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}" }
+end
+
+# rubocop:disable Metrics/MethodLength
+def create_person
+  print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+  person_type = gets.chomp
+
+  if person_type != '1' && person_type != '2'
+    puts 'Invalid option'
+    return
+  end
+
+  print 'Age: '
+  age = gets.chomp
+
+  print 'Name: '
+  name = gets.chomp
+
+  case person_type
+  when '1'
+    print 'Has parent permission? [Y/N]: '
+    parent_permission = gets.chomp
+    parent_permission = parent_permission.downcase == 'y'
+
+    person = Student.new(age, name: name, parent_permission: parent_permission)
+
+  when '2'
+    print 'Specialization: '
+    specialization = gets.chomp
+
+    person = Teacher.new(age, specialization: specialization, name: name)
+  end
+
+  @people << person
+
+  puts 'Person created successfully'
+end
+# rubocop:enable Metrics/MethodLength
+
+def create_book
+  print 'Title: '
+  title = gets.chomp
+
+  print 'Author: '
+  author = gets.chomp
+
+  @books << Book.new(title, author)
+  puts 'Book created successfully'
+end
+
+def create_rental
+  puts 'Select a book from the following list by number'
+  @books.each_with_index { |book, index| puts "#{index}) Title: \"#{book.title}\", Author: #{book.author}" }
+  book_index = gets.chomp.to_i
+
+  puts
+
+  puts 'Select a person from the following list by number (not id)'
+  # rubocop:disable Metrics/LineLength
```

Comment:
- [ ] Your app works fine but I noticed that you disabled **Rubocop** a lot is not a good practice. Please remove all the **Rubocp disable** statements in your codebase and try to fix all the errors you are getting. Because ` Rubocop` helps developers to write clean Ruby code and adapt best practices when writing the code.  :+1: